nengo/params.py

```diff
@@ -416,7 +417,7 @@ def validate(self, instance, function_info):
         super(FunctionParam, self).validate(instance, function)
 
 
-class FrozenObject(object):
+class FrozenObject(with_metaclass(DocstringInheritor)):
```

Comment:
It also seems to work the other ways, so I'm not sure what I was doing wrong before...